applicationinsights/data-plane/samples/src/main/java/com/microsoft/azure/applicationinsights/query/samples/ApplicationInsightsClientExample.java

```diff
@@ -0,0 +1,75 @@
+package com.demo.app;
```

Comment:
package is incorrect - should match folder structure `com.microsoft.azure.applicationinsights.query.samples`.

Reply:
fixed